rules/static_test.go

```diff
@@ -125,7 +126,7 @@ var _ = Describe("Static", func() {
 							{Action: JumpAction{Target: ChainDispatchFromHostEndpoint}},
 							{
 								Match:   Match().MarkSet(0x10),
-								Action:  AcceptAction{},
```

Comment:
This `Describe` block is meant to test the default config, please can you add a new describe with the updated config and a new test?